examples/step-14/step-14.cc

```diff
@@ -2188,19 +2192,24 @@ namespace Step14
              ++face_no)
           face_integrals[cell->face(face_no)] = -1e20;
 
+      auto worker = [this,
+                     &error_indicators,
+                     &face_integrals](const active_cell_iterator & cell,
+                                      WeightedResidualScratchData &scratch_data,
+                                      WeightedResidualCopyData &   copy_data) {
+        this->estimate_on_one_cell(
+          cell, scratch_data, copy_data, error_indicators, face_integrals);
+      };
+
+      auto copier = std::function<void(const WeightedResidualCopyData &)>();
```

Comment:
Let's make it extra clear :-)
```suggestion
      auto do_nothing_copier = std::function<void(const WeightedResidualCopyData &)>();
```